src/ui/components/RedisStats.tsx

```diff
@@ -40,10 +32,12 @@ const getMemoryUsage = (
       ).toFixed(2)}%`
     : formatBytes(parseInt(used_memory, 10))
 
-export const RedisStats = ({ stats }: { stats: ValidMetrics }) => {
+export const RedisStats = ({ stats }: { stats: Partial<ValidMetrics> }) => {
   const {
     redis_version,
-    used_memory,
+    // REVIEW: the API gives no guarantee these fields will be there.
+    //  Is it fine to use '0' as the fallback here?
```

Comment:
Yeah, that's probably a good idea.

Ideally the API would ensure that a default is always returned, but that's a task for the future 🙂

Reply:
used memory = 0 can mean different stuff than "we couldn't retrieve it", maybe that should be more clear, at least a `-`

Reply:
I can change the code that shows up this number in the screen rather than adding the default case